src/integration-test/groovy/uk/ac/ox/softeng/maurodatamapper/plugins/fhir/datamodel/provider/importer/FhirDataModelImporterProviderServiceSpec.groovy

```diff
@@ -502,6 +502,7 @@ class FhirDataModelImporterProviderServiceSpec extends BaseIntegrationSpec {
         assert dataModel.label == entryId
         dataModel.folder = folder
         dataModelService.validate(dataModel)
+        fhirDataModelImporterProviderService.updateImportedModelFromParameters(dataModel, parameters, false)
```

Comment:
Just FYI I think this should already be called through the parent class ModelImporterProviderService (in `importDomain`).

Reply:
Added this line on the basis that the current tests should fall foul of the parsing error (because they use data harvested from the same source, with the same format in the `date` field), but don't.

A breakpoint placed inside the `updateImportedModelFromParameter` method does not trip when debugging the tests in `FhirDataModelImporterProviderServiceSpec`.

- Placed a `throw IllegalArgumentException("date parse")` just before the date parse so all calls will fail
- This results in 9 failures, none of them for importing `StructureDefinition` (all code sets and terminologies)

Reply:
@awilkins, sorry, I had not noticed that this was in the `Spec` file not the actual importer! Yes I agree that makes sense in the tests :)